tasks/test.yml

```diff
@@ -257,10 +257,12 @@ tasks:
     desc: "Runs the site installer for the Firefox container"
     cmds:
       - ./vendor/bin/drush --uri=https://drupal_firefox --yes site:install --existing-config
+      - mkdir -p /var/www/html/web/sites/firefox/files/private
```

Comment:
Hard coding the path to the private directory doesn't feel great to me, but I put it in to demonstrate what I was running into.

Reply:
I was thinking about this, and won't this path change based on the site's settings.php? It wouldn't matter for normal functional tests, but for nightwatch / DTT this file would need to be _symlinked_ to whatever that directory is.